ros_ws/src/autonomous/q_learning/launch/autonomous_driving.launch

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<launch>
-<!-- Launches all necessary nodes that let the car drive autonomously. -->
-    <node
-      respawn="true"
-      pkg="q_learning"
-      type="drive.py"
-      name="q_learning_driver"
-      output="screen" >
-    </node>
-</launch>
```

Comment:
Why was this deleted?

Reply:
I thought this was a file used for debugging purposes during development that erroneously made it's way onto the master. Sorry, I'll restore it.